### PR TITLE
Added Greek translations

### LINKED ISSUE #362: Greek language support
At the moment there is no Greek language support. As a native Greek speaker I would be happy to do the translation needed.

### PR DESCRIPTION
**Pull request by karafyllidis**
Added Greek translations - closes #362 

Signed-off-by: Tasos Karafyllidis karafyllidis@gmail.com (github: karafyllidis)

### CODE REVIEW

**Comment by mueller-ma** [comment]
"about_links_list" is missing. Cannot speak about the translation itself, but it builds and the language code is correct. 👍

File: mobile/src/main/res/values-el/strings.xml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    <string name="settings_openhab_url_summary">Ρύθμιση διεύθυνσης openHAB με το χέρι. Απενεργοποιεί την αυτόματη εύρεση αν έχει ήδη ρυθμιστεί.</string>
+    <string name="settings_openhab_username">Όνομα χρήστη</string>
+    <string name="settings_openhab_username_summary">Το όνομα χρήστη (username) του openHAB</string>
+    <string name="settings_ringtone">Ήχος κλήσης</string>
+    <string name="theme_name_black">Μαύρο/AMOLED</string>
+    <string name="theme_name_dark">Σκούρο</string>
+    <string name="theme_name_light">Ανοιχτόχρωμο</string>
+    <string name="theme_value_black">Μαύρο</string>
+    <string name="theme_value_dark">Μαύρο</string>
+    <string name="theme_value_light">Ανοιχτόχρωμο</string>
```
Comment (mueller-ma) [inline mobile/src/main/res/values-el/strings.xml]: Please remove the three "theme_value_..." strings. They will become obsolete when #350 is merged

**Comment by karafyllidis** [comment]
Thank you for the remarks. I have made the necessary changes as instructed.

**Comment by mueller-ma** [comment]
@digitaldan Can be merged

**Comment by digitaldan** [comment]
thanks!

**Pull request merged**